README.md

```diff
@@ -88,3 +87,26 @@ finalize_plot(title = "Regional population and labor force participation",
 ```
 
 <img src="man/figures/README-finalize-1.png" width="672" style="display: block; margin: auto;" />
+
+## Additional reading
+
+While this package is designed to make the application of CMAP design
+standards to plots relatively easy, developing a professional, finished
+plot in R will require a decent familiarity with the grammar of
+[ggplot2](ggplot2.tidyverse.org/). Excellent resources in this category
+already exist:
+
+  - The [R graphics cookbook](https://r-graphics.org/) provides
+    accessible examples of how to make almost [any
+    type](https://r-graphics.org/recipe-miscgraph-vectorfield) of plot,
+    as well as how to modify things like [limits,
+    scales](https://r-graphics.org/recipe-axes-range), [coordinate
+    systems](https://r-graphics.org/recipe-axes-polar), and
+    [facets](https://r-graphics.org/recipe-facet-basic).
+
+  - The [ggplot2 book](https://ggplot2-book.org/) delves deeper into why
+    and how ggplot2 works the way it does, also with distinct chapters
+    on topics like
+    [scales](https://ggplot2-book.org/scales-guides.html), [coordinate
+    systems](https://ggplot2-book.org/coord.html),
+    [facets](https://ggplot2-book.org/facet.html), etc.
```

Comment:
Consider also adding links to [R for Data Science](https://r4ds.had.co.nz) and/or the official [ggplot2 package website](https://ggplot2.tidyverse.org)?